Coral/TileEditor/interactive.js

```diff
@@ -1,6 +1,6 @@
-import { Flector } from "../pga2d/versors.js";
+import { Flector } from "../../pga2d/versors.js";
 
-export const SELECT_RADIUS = 8;
+export const SELECT_RADIUS = 16;
```

Comment:
This is too large, I'm noticing the mouse is selecting things in a non-obvious way.